HBFA/UefiHostFuzzTestCasePkg/TestCase/OvmfPkg/Library/CcExitLib/TestParseMmioExitInstructions.c

```diff
@@ -21,6 +21,7 @@
 #include <Library/MemoryAllocationLib.h>
 #include <Library/PcdLib.h>
 #include <Library/CcExitLib.h>
+#include <Library/CcExitLib/CcExitVeHandler.c>  // TODO: temporary hack for static functions
```

Comment:
Remove todo

Reply:
fixed in [0148b9e](https://github.com/tianocore/edk2-staging/pull/386/commits/0148b9e1fc1d0947610f4bb1da22a6b373d9258e)